weldx/schemas/weldx.bam.de/weldx/core/media_file-0.1.0.yaml

```diff
@@ -0,0 +1,38 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://weldx.bam.de/weldx/schemas/core/media_file-0.1.0"
+
+title: |
+  Schema for a media file (time-based and single shot).
+description: |
+  This schema describes a media file by compiling all its meta data.
+  Optionally, the whole file can also be stored in the
+  binary block of an asdf file.
+
+examples:
+  -
+    - foo
+    - bar
+
+type: object
+properties:
+  file:
+    description: |
+      The basic file information/content.
+    tag: "asdf://weldx.bam.de/weldx/tags/core/file-0.1.1"
```

Comment:
we can (and probably should) add the requirement here that `mimetype` of this file should follow `pattern: video/*`

But we can add that later